test/fixtures/access-control/server/server.js

```diff
@@ -5,7 +5,7 @@
 
 var loopback = require('../../../..');
 var boot = require('loopback-boot');
-var app = module.exports = loopback({ localRegistry: true });
+var app = module.exports = loopback({ localRegistry: true, loadBuiltinModels: true });
```

Comment:
FWIW, this should not be necessary, loopback-boot loads builtin models as needed.